### PR TITLE
feat: adding srs APIs, refactoring kas APIs

### PR DESCRIPTION
**Pull request by secondsun**
Provides `Refactor the current Kafka Managed Services API to a new package namespace (.kas)` and 
`Add the SRS Open APIs (.srs) ` from https://github.com/redhat-developer/app-services-operator/issues/217

### CODE REVIEW

File: source/rhoas/src/main/java/com/openshift/cloud/beans/KafkaApiClient.java [hunk 1]
```diff
@@ -1,14 +1,14 @@
 package com.openshift.cloud.beans;
 
-import com.openshift.cloud.ApiClient;
-import com.openshift.cloud.ApiException;
-import com.openshift.cloud.Configuration;
-import com.openshift.cloud.api.DefaultApi;
+import com.openshift.cloud.api.ApiClient;
+import com.openshift.cloud.api.ApiException;
```
Comment (wtrocki) [inline source/rhoas/src/main/java/com/openshift/cloud/beans/KafkaApiClient.java:4]: our of curiosity - why this is not namespaced?

File: source/openapi/pom.xml [hunk 2]
```diff
@@ -128,7 +128,31 @@
                             <generateApiTests>false</generateApiTests>
                             <configOptions>
                                 <modelPackage>com.openshift.cloud.api.models</modelPackage>
-                                <apiPackage>com.openshift.cloud.api</apiPackage>
+                                <apiPackage>com.openshift.cloud.api.kas</apiPackage>
```
Comment (wtrocki) [inline source/openapi/pom.xml:131]: Is model package ok here. it is the same for both

File: source/rhoas/src/main/java/com/openshift/cloud/beans/KafkaApiClient.java (hunk 1 above)
Comment (secondsun) [reply source/rhoas/src/main/java/com/openshift/cloud/beans/KafkaApiClient.java:4]: They're common classes that are shared by both implementations. The ApiClient handles Http, suth, etc and ApiException is an execption.

File: source/openapi/pom.xml (hunk 2 above)
Comment (secondsun) [reply source/openapi/pom.xml:131]: Good catch

File: source/rhoas/src/main/java/com/openshift/cloud/beans/KafkaApiClient.java (hunk 1 above)
Comment (wtrocki) [reply source/rhoas/src/main/java/com/openshift/cloud/beans/KafkaApiClient.java:4]: Ok.. So this is quite well done. I see duplication in golang so need to figure out if there is the same option for golang genator to have them separately.

CC @craicoverflow

**Review by wtrocki — APPROVED** [review]

**Pull request merged**